Language/Structure/Compound Operators/increment.adoc

```diff
@@ -24,8 +24,11 @@ Increments the value of a variable by 1.
 
 [float]
 === Syntax
-`x++;  // increment x by one and returns the old value of x` +
-`++x;  // increment x by one and returns the new value of x`
+[source,arduino]
+----
+x++;  // increment x by one and returns the old value of x
+++x;  // increment x by one and returns the new value of x
+----
```

Comment:
```suggestion
`x{plus}{plus};  // increment x by one and returns the old value of x` +
`{plus}{plus}x;  // increment x by one and returns the new value of x`
```
I think this is a better approach because it makes it more obvious (to me at least) why the markup needs to be this way.

With your appoach, it's not clear why it doesn't follow the established markup convention for the syntax sections, which could lead to some confusion for future editors or even the fix being undone.